### ISSUE TITLE
Past events can not be hidden 

### ISSUE THREAD

**Issue by angeloprevitali** [daCyberpunk/dated_news#10]
In the Plug-In EXT:news the filed of "timelimit" if you add "-1 hour" as it is the default value for time limitation the FE get no more results - even if there are future events.

![1](https://user-images.githubusercontent.com/17588253/37341635-67c8e3fa-26c2-11e8-95c8-e88965c12f1a.png)

Can you fix this?

Thanks.

Regards,
Angelo

**Comment by angeloprevitali** [comment]
We use TYPO3 CMS V8.7.10 + EXT:news 6.3.0 and EXT:dated_news Version 5.1.1

**Comment by bocianl** [comment]
Yes, for me is also an important possibility to hide past events.
